.github/workflows/chatflows.yml

```diff
@@ -0,0 +1,35 @@
+name: Chatflows Tests
+
+on:
+  release:
+    types: [ prereleased ]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Gathering deps
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y git python3-pip python3-venv python3-setuptools tmux redis restic nginx cron wireguard
+          sudo service redis stop
+          sudo service nginx stop
+          git clone https://github.com/threefoldtech/js-sdk.git
+          cd js-sdk
+          sudo pip3 install poetry
+          sudo poetry run pip3 install pytest 
+      - name: Install
+        run: |
+          cd js-sdk
+          sudo poetry install
+      - name: Run tests
+        env:
+          TNAME: ${{ secrets.TNAME }}
+          EMAIL: ${{ secrets.EMAIL }}
+          WORDS: ${{ secrets.WORDS }}
+          WALLET_SECRET: ${{ secrets.WALLET_SECRET }}
+        run: |
+          cd js-sdk
+          sudo --preserve-env=TNAME --preserve-env=EMAIL --preserve-env=WORDS --preserve-env=WALLET_SECRET \
```

Comment:
i think it should be $WORDS, and $WALLET_SECRET also why not do that directly via the ${{ secrets.WORDS }} directly in the run command ?

Reply:
I can't use Github secrets while using `sudo`, so I take them to `sudo` using their names without `$`